Streamline sizes of several UI panels

dragunoff opened the pull request:
I took a pass at resizing and adjusting the white space in many of the UI panels. They fit in two size categories now:
- Large (Multiplayer Browser, Lobby, Asset Browser, Replay Browser)
- Medium (Mission Browser, Settings - after #19677)
- Small (Credits, Music Player) (same height as the medium size)

I have not yet touched several panels (Game Save/Load, In-game Info). I plan to address them in a follow up because the size of the changes here is starting to get intimidating. Some additional polish may be needed but I also want to leave that to follow ups or at least after #19677 and #19461).

Things are still not quite consistent (especially wrt to white-space) as it is quite difficult to layout everything with absolute positioning but nevertheless this PR makes great progress towards #16596

It's best to review by opening two clients and switching between the windows to see the differences and how the panel dimensions line up.

Supersedes #19680

dragunoff committed:
Make size and white-space of "Music" and "Credits" panels consistent

dragunoff committed:
Make size and white-space of "Asset Browser" panel consistent

dragunoff committed:
Make size and white-space of "Replay Browser" panel consistent

dragunoff committed:
Make size and white-space of "Mission Browser" panel consistent

dragunoff committed:
Adjust size, position and white-space of several panels

pchote commented on mods/cnc/chrome/assetbrowser.yaml:
```diff
@@ -1,9 +1,9 @@
-	Y: (WINDOW_BOTTOM - HEIGHT) / 2
-	Width: 695
-	Height: 435
+	Y: (WINDOW_BOTTOM - 560) / 2
```
The other TD panels were changed to use `WINDOW_BOTTOM - HEIGHT` - why change this one away from that?

dragunoff replied:
Good catch. This occurred to me as well. There is also an issue with the buttons at the bottom of panels as in some cases they are included in the panel height and in others not. I will look at what works best here and try to tie it all together.

pchote replied:
My original idea wrt the y alignment was that the panel should be centered, ignoring the title on the top or the buttons below. We don't necessarily need to keep that provided we are consistent.

pchote reviewed (commented):
Looks good overall.

* Can we please use this opportunity to fix the incorrect top margin in the TD replay browser? The lobby window can be used as an example for the spacing.
* The column spacing between the sidebar and main panel in the common asset browser looks too narrow compared to the lobby/mission browser/multiplayer browser?

dragunoff committed:
Make size and white-space of game save/load panel more consistent

dragunoff commented:
Update: addressed comments so far and touched up a few other places that I noticed.

I also made a quick pass over the game save/load panel:
![Screenshot-20211007210444-1282x807](https://user-images.githubusercontent.com/1355810/136439935-1d919f74-5df8-46a1-a744-363819eb68cd.png)
![Screenshot-20211007210533-1282x807](https://user-images.githubusercontent.com/1355810/136439937-789ac520-5c03-4d1a-ad09-7dffd704c952.png)

This panel is the odd one out and I think it needs a redesign but that is definitely work for a follow up.

reaperrr commented:
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/ba71bfffe2da0ffffee1e2c535e787d62de8c40d...25f71ff58e3535bbfb2f18031eb155d170b43be5
